en/docs/webhooks.md

```diff
@@ -1,8 +1,14 @@
 # Webhooks
 
-A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response. Choreo allows you to create webhooks for platforms such as Github, Slack, and Google Calendar using the integrated pro-code or low-code editor. For example, you can design a webhook to notify a Github user via mail each time an issue is assigned to him/her.
+A webhook directs information from one application to another in real-time and triggers the receiving application to execute a specific process in response. Choreo allows you to create webhooks for platforms such as Github, Slack, and Google Calendar using the integrated pro-code or low-code editor.
```

Comment:
Should we capitalize the W in the term Webhook as it is a name of a component in Choreo?

Reply:
Used capital W when it is referred to as the "Webhook component".